File: Py/Card/Card.py
class Card(object):
    def __init__(self, name, desc):
        self.Name = name
        self.Desc = desc
        self.Type = "None"

    def OnHand(self):
        return 1

    def Play(self):
        return 1

    def Aban(self):
        return 1

    def __str__(self):
        return "[{},{},{}]".format(self.Name, self.Desc, self.Type)

File: C++/Card/Card.cpp
//
// Created by DOEMsy on 2021/1/2.
//

#include "Card.h"

Card::Card() {
    this->Type = CardTypeNULL;
}

Card::Card(const std::string &Name, const std::string &Desc) {
    this->Name = Name;
    this->Desc = Desc;
    this->Type = CardTypeNULL;
}

Card::~Card() {

}


